Error: Cannot find module 'core-js/modules/es.symbol'

<!-- IF YOU DO NOT FILL OUT THE FOLLOWING FORM YOUR ISSUE WILL AUTOMATICALLY BE CLOSED!!! -->

# Expected Behavior
The module doesn't cause the program to crash

<!-- Please describe the behavior you are expecting -->

# Current Behavior
The module makes the program crash

<!-- What is the current behavior? -->

# Failure Information (for bugs)

> Error: Cannot find module 'core-js/modules/es.symbol'
Require stack:
> - C:\Users\User\Desktop\bots\node_modules\@jimp\custom\dist\index.js
> - C:\Users\User\Desktop\bots\node_modules\jimp\dist\index.js
> - C:\Users\User\Desktop\bots\test2.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:17)
    at Function.Module._load (internal/modules/cjs/loader.js:859:27)
    at Module.require (internal/modules/cjs/loader.js:1028:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\User\Desktop\bots\node_modules\@jimp\custom\dist\index.js:7:1)
    at Module._compile (internal/modules/cjs/loader.js:1139:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1159:10)
    at Module.load (internal/modules/cjs/loader.js:988:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1028:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\User\\Desktop\\bots\\node_modules\\@jimp\\custom\\dist\\index.js',
    'C:\\Users\\User\\Desktop\\bots\\node_modules\\jimp\\dist\\index.js',
    'C:\\Users\\User\\Desktop\\bots\\test2.js'
  ]

<!-- Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template. -->

## Steps to Reproduce

`const jimp = require('jimp');`

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->
None
## Context

<!-- Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions. -->

- Jimp Version: v0.11.0
- Operating System: Win 10/Win Server 2012
- Node version: 13.x

## Failure Logs
Provided above

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -44,22 +44,6 @@ jobs:
           name: Test Types
           command: yarn tsTest:custom && yarn tsTest:main
 
-  build-node6.14:
-    working_directory: ~/jimp-6.14
-    docker:
-      - image: circleci/node:6.14-browsers
-    steps:
-      - checkout
-      - run:
-          name: Install Dependencies
-          command: yarn install --frozen-lockfile --network-timeout 100000  --ignore-engines
-      - run:
-          name: Build Packages
-          command: yarn build
-      - run:
-          name: Test
-          command: yarn test --ci
-
   build-node8:
     <<: *defaults
     steps:
@@ -117,11 +101,6 @@ workflows:
       - test-types:
           requires:
             - install
-
-      - build-node6.14:
-          requires:
-            - lint
-            - test-types
 
       - build-node8:
           requires:

--- a/babel.config.js
+++ b/babel.config.js
@@ -3,13 +3,7 @@ module.exports = api => {
 
   return {
     presets: [
-      [
-        '@babel/preset-env',
-        {
-          useBuiltIns: 'usage',
-          corejs: 3
-        }
-      ]
+      '@babel/preset-env',
     ],
 
     plugins: [

